### PR TITLE
[DEST-931] Add setting for custom ad asset ID property name

### PR DESCRIPTION
**Pull request by CarlosMecha**

### CODE REVIEW

**Comment by brennan** [comment]
@CarlosMecha Brie and I are going to make some changes I've outlined here: https://paper.dropbox.com/doc/Nielsen-DCR-asset_id-mapping-guide--Ah9O_NMOhsuGvuV1697iLeHnAg-qLq8RKcgiK1yjZh8R0wgx

tl;dr depending on the specced video event, a default content asset id property can be either `contentAssetId` or `assetId`, and a default ad asset id property can be either `adAssetId` or `assetId`. The logic should take this nuance into account.

I am ok being in charge of updating the Android code today while you get better! :) Let me know if you have any questions.

File: src/main/java/com/segment/analytics/android/integrations/nielsendcr/NielsenDCRIntegrationFactory.java [hunk 1]
```diff
@@ -85,6 +85,10 @@ static NielsenDCRIntegrationFactory create() {
       if (assetIdPropertyName != null && !assetIdPropertyName.isEmpty()) {
```
Comment (briemcnally) [inline src/main/java/com/segment/analytics/android/integrations/nielsendcr/NielsenDCRIntegrationFactory.java]: We will want to change to `contentAssetIdPropertyName`.

File: src/main/java/com/segment/analytics/android/integrations/nielsendcr/NielsenDCRIntegration.java [hunk 2]
```diff
@@ -33,13 +33,15 @@
...
+    String adAssetIdPropertyName;
     String assetIdPropertyName;
     String clientIdPropertyName;
     String subbrandPropertyName;
     String contentLengthPropertyName;
 
     Settings() {
       // Null by default
+      adAssetIdPropertyName = null;
       assetIdPropertyName = null;
```
Comment (briemcnally) [inline src/main/java/com/segment/analytics/android/integrations/nielsendcr/NielsenDCRIntegration.java]: We will want to change this to `contentAssetIdPropertyName`

**Comment by brennan** [comment]
Closing b/c this feature was added in another branch, which has already been merged and released.

**Pull request closed without merging**